pkgs/games/assaultcube/default.nix

```diff
@@ -0,0 +1,80 @@
+{ fetchgit, stdenv, makeDesktopItem, curl, enet, openal, pkgconfig, libogg,
+  libvorbis, SDL, SDL_image, SDL_mixer, makeWrapper, zlib,
+  client ? true, server ? true }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+
+  # master branch has legacy (1.2.0.2) protocol 1201 and gcc 6 fix.
+  branch = "master";
```

Comment:
Could we have some sort of version attribute here, appended to name? Perhaps just the date of the commit in question.